CHANGELOG.md

```diff
@@ -7,6 +7,18 @@
 - (Internal) Extract Android Profiler and Measurements for Hybrid SDKs ([#3016](https://github.com/getsentry/sentry-java/pull/3016))
 - Ensure DSN uses http/https protocol ([#3044](https://github.com/getsentry/sentry-java/pull/3044))
 
+## 6.33.2-beta.1
```

Comment:
`m`: should this rather/also be in `## Unreleased`?

Reply:
Just talked with Roman about this, lets dup it 👍